openprescribing/frontend/views/bookmark_utils.py

```diff
@@ -260,6 +260,8 @@ def __init__(self, org, interesting_saving=1000, interesting_change_window=12):
             self.measure_filter_for_org = {"pcn": org, "practice": None}
         elif isinstance(org, PCT):
             self.measure_filter_for_org = {"pct": org, "practice": None}
+        elif isinstance(org, STP):
+            self.measure_filter_for_org = {"stp": org, "practice": None, "pct": None}
```

Comment:
Should we be adding `"stp": None` to the branches above?

Reply:
No, that would break it because it would exclude all MeasureValues for any organisation at STP level or less in the hierarchy. It's only organisation types _below_ the target organisation for which you need to add `None` filters.

I didn't remember any of this stuff but fortunately, as a side effect of doing some Pandas reshaping, the tests blow up with an error if you don't get your organisation filters right.

Reply:
Oh, yuck.